internal/bundle/helpers.go

```diff
@@ -0,0 +1,70 @@
+package bundle
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/internal"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/databricks/cli/libs/template"
+)
+
+func initTestTemplate(t *testing.T, templateName string, config map[string]any) (string, error) {
+	templateRoot := filepath.Join("bundles", templateName)
+
+	bundleRoot := t.TempDir()
+	configFilePath, err := writeConfigFile(t, config)
+	if err != nil {
+		return "", err
+	}
+
+	ctx := root.SetWorkspaceClient(context.Background(), nil)
```

Comment:
It would be nice if we have a general test helper that sets up a workspace client mock.

Then we don't need to do nil checks in our code and we can mock out any calls the underlying code would make.

In this case though, wouldn't we authenticate automatically with the workspace configured for this integration test? If so, we don't need to set this to nil at all.

Reply:
This is done to avoid errors at this line https://github.com/databricks/cli/blob/c0ebfb8101700d6e6300954c2efcf2077c690e01/libs/template/helpers.go#L33

We just need to have something set in the context for workspace client so the code won't fail. In the rest of the test we indeed automatically authenticate workspace